dist/pivot.da.min.js

```diff
@@ -0,0 +1,61 @@
+(function () {
```

Comment:
Thanks for this. I will try to find time to port it into Coffeescript for inclusion.